docs/user-guide/aws-examples/eks.rst

```diff
@@ -0,0 +1,215 @@
+======================================
+Running NeMo Curator on AWS EKS
+======================================
+
+Prerequisuites:
+
+* EKS Cluster:
+    * `Dask Operator <https://kubernetes.dask.org/en/latest/operator_installation.html>`__
+    * If self managed node group is created with ubuntu worker nodes then install GPU operator.
+        `GPU operator <https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/getting-started.html>`__
+    * If EKS managed node group is created with Amazon Linux 2 worker nodes then install Nvidia device plugin.
+        `Nvidia Device Plugin installation <https://docs.aws.amazon.com/eks/latest/userguide/eks-optimized-ami.html>`__
+    * `Dask Operator <https://kubernetes.dask.org/en/latest/operator_installation.html>`__
+* Storage:
+    * `EFS for EKS <https://github.com/kubernetes-sigs/aws-efs-csi-driver/blob/master/docs/efs-create-filesystem.md>`__ (setup by Kubernetes cluster admin)
+
+Create a Storage Class for AWS EFS
+----------------------------------
+
+.. code-block:: yaml
+
+   cat <<EOF | kubectl apply -f -
+   kind: StorageClass
+   apiVersion: storage.k8s.io/v1
+   metadata:
+     name: efs-sc
+   provisioner: efs.csi.aws.com
+   parameters:
+     provisioningMode: efs-ap
+     fileSystemId: ${FileSystemId}  # Replace with your actual FileSystemId
+     directoryPerms: "700"
+   EOF
+
+In the above YAML:
+
+- Replace `${FileSystemId}` with the actual EFS FileSystemId from AWS.
+- This definition sets up a StorageClass named `efs-sc` that provisions PersistentVolumes using the AWS EFS CSI Driver.
+
+
+PVC Definition
+--------------------------------
+Now, we can use the storage class created in the previous step to create a PVC.
+
+.. code-block:: yaml
+
+   cat <<EOF | kubectl apply -f -
+   apiVersion: v1
+   kind: PersistentVolumeClaim
+   metadata:
+     name: nemo-workspace
+   spec:
+     accessModes:
+       - ReadWriteMany
+     storageClassName: efs-sc
+     resources:
+       requests:
+         storage: 150Gi
```

Comment:
Not too familiar with EKS, but do should this be `150GiB` instead?

Reply:
Referred these examples: https://github.com/kubernetes-sigs/aws-efs-csi-driver/blob/master/examples/kubernetes/dynamic_provisioning/specs/pod.yaml and it works.

Reply:
Great, thanks for checking.